### ISSUE TITLE
Kindle again

### ISSUE THREAD

**Issue by library-user** [apprenticeharper/DeDRM_tools#2207]
I have a KOBO Libra e-reader (with which I am blissfully happy).
At the moment, a newly published book I want to read is only available in electronic form via Amazon Kindle.
Can I, perhaps using some mechanism in Calibre, purchase the book and get it on my Kobo, please?
Thanks
Brian (library-user)

**Comment by ElleKayEm** [comment]
If you mean newly published this year, probably not unless you also have an E-ink Kindle or maybe if it's a comic or manga.  See this post:  https://www.mobileread.com/forums/showthread.php?t=283371

You can find the latest DeDRM plugin here: https://github.com/noDRM/DeDRM_tools/releases/tag/v10.0.3

**Comment by NiQ1** [comment]
> If you mean newly published this year, probably not unless you also have an E-ink Kindle or maybe if it's a comic or manga. See this post: https://www.mobileread.com/forums/showthread.php?t=283371
> 
> You can find the latest DeDRM plugin here: https://github.com/noDRM/DeDRM_tools/releases/tag/v10.0.3

Would a cheap 2nd hand D00901 from eBay do the trick?

**Comment by ElleKayEm** [comment]
The Kindle Keyboard would technically work, but you'd get MOBI files instead of AZW3.  If you're going to do something older, the Kindle Touch is as old as you'd probably want to do.  That and the Paperwhite 1 are new enough to get AZW3, but old enough they don't get KFX format.  And their firmware is no longer being updated.  But even the current models will give you access to downloading AZW3 files from the website which you can decrypt (unless its one of a few books that only come in KFX format).  At least that's the situation currently.  Who knows what Amazon might change or cut off next.

**Comment by NiQ1** [comment]
> The Kindle Keyboard would technically work, but you'd get MOBI files instead of AZW3. If you're going to do something older, the Kindle Touch is as old as you'd probably want to do. That and the Paperwhite 1 are new enough to get AZW3, but old enough they don't get KFX format. And their firmware is no longer being updated. But even the current models will give you access to downloading AZW3 files from the website which you can decrypt (unless its one of a few books that only come in KFX format). At least that's the situation currently. Who knows what Amazon might change or cut off next.

Would D01200 work then?

If not I'd really appreciate if you post the optimal model number. I've never really used physical readers before.

**Comment by ElleKayEm** [comment]
Yes that would work.  I have one of those myself.  You can both download from the website and get files directly from the device that can be decrypted.

Here are some thoughts from someone who has more knowledge on it than me:  https://www.mobileread.com/forums/showpost.php?p=4288334&postcount=192

And here: https://www.mobileread.com/forums/showpost.php?p=4289735&postcount=464

(Apparently I was mistaken about 3rd generation Kindle Keyboards only getting mobi.  Looks like that is just 1st and 2nd generation models.)

**Comment by library-user** [comment]
Yay! No longer any need to worry about Kindle. An EPub version of the  book has just been released. Solves all problems.
Thanks to everyone for their thoughts and help.
Library-User

**Comment by ElleKayEm** [comment]
If you're all set, you can close the issue.
